homeassistant/components/vesync/strings.json

```diff
@@ -54,6 +54,9 @@
     "number": {
       "mist_level": {
         "name": "Mist level"
+      },
+      "night_light_brightness": {
+        "name": "Night brightness level"
```

Comment:
I think it should have the word light in it.

Reply:
I had picked this because the card only is only able to show part of that string.

![image](https://github.com/user-attachments/assets/abc09ca0-36f2-49e9-98a3-f712d3e4b08f)

I think "Nightlight level" feels the best.